src/main/java/de/tum/in/www1/artemis/service/connectors/localvc/LocalVCPrePushHook.java

```diff
@@ -44,6 +52,21 @@ public void onPreReceive(ReceivePack receivePack, Collection<ReceiveCommand> com
 
         Repository repository = receivePack.getRepository();
 
+        String mainBranchName;
+        try {
+            mainBranchName = localVCServletService.getMainBranchOfRepository(repository);
+        }
+        catch (LocalVCInternalException e) {
+            command.setResult(ReceiveCommand.Result.REJECTED_OTHER_REASON, "An error occurred while checking the branch.");
+            return;
+        }
+
+        // Reject pushes to anything other than the default branch.
+        if (!command.getRefName().equals("refs/heads/" + mainBranchName)) {
```

Comment:
Further aligning the naming mentioned by Stroh.
```suggestion
        String defaultBranchName;
        try {
            defaultBranchName = localVCServletService.getDefaultBranchOfRepository(repository);
        }
        catch (LocalVCInternalException e) {
            command.setResult(ReceiveCommand.Result.REJECTED_OTHER_REASON, "An error occurred while checking the branch.");
            return;
        }

        // Reject pushes to anything other than the default branch.
        if (!command.getRefName().equals("refs/heads/" + defaultBranchName)) {
```